kernel/mem/paging.cpp

```diff
@@ -50,6 +56,11 @@ static inline void map_kernel_page_table(uint32_t pd_idx, page_table_t *table);
 static inline void set_page_dir(uint32_t page_directory);
 static inline void paging_enable();
 static inline void paging_disable();
+static void paging_args_cb(const char* arg);
+static CONSTRUCTOR void paging_args_init();
```

Comment:
This function isn't defined anywhere, remove the declaration

Reply:
Thanks, missed that in the refactor.